smartsim/_core/mli/infrastructure/control/request_dispatcher.py

```diff
@@ -493,10 +493,8 @@ def flush_requests(self) -> None:
                 if queue.ready:
                     self._perf_timer.measure_time("find_queue")
                     try:
-                        batch = RequestBatch(
-                            requests=queue.flush(),
-                            inputs=None,
-                            model_id=queue.model_id,
+                        batch = RequestBatch.from_requests(
```

Comment:
Something feels off about passing the requests and the inputs in here. It might be the type of the input...

If this method accepts a `TransformInputRequest`, I expect that input to have gone through the workers transformation stage.... That would be something like this, but the worker obviously isn't available:

```
my_requests  = request.flush()
my_transformed_data = [worker.transform(r) for r in my_requests]
batch = ReqBatch.from(my_transformed_data)
```

- Maybe you really want to pass a type other than transformed input? A `FetchInputResult` implies that this is newly fetched data and fits better (since we're taking raw data from the request). That might reduce the work you need to do and allow the transform from raw data to only live in a worker?

Reply:
After our conversation, I've removed the `inputs` parameter from `RequestBatch.from_requests`!